app/account/hooks/useUsersGet.ts

```diff
@@ -0,0 +1,22 @@
+import { useUserId } from '@/providers/session-provider'
+import { useGetUserQuery } from '@generated/api'
+
+export const useUserGet = () => {
```

Comment:
If you create this,   could you modify the account details pages too for consistency?

Reply:
I changed it!